test/unit/extras/command_wrapper_test.rb

```diff
@@ -125,10 +125,10 @@ def mock_connect_result(stderr, exit_status)
     end
 
     it "error message for sudo password required" do
-      backend.stubs(:run_command).returns(mock_connect_result("sudo: no tty present and no askpass program specified", 1))
+      backend.stubs(:run_command).returns(mock_connect_result("sudo: no tty present and no program specified", 1))
       lc = cls.new(backend, { sudo: true })
       err = _ { lc.verify! }.must_raise Train::UserError
-      _(err.message).must_match(/Sudo requires a password, please configure it./)
+      _(err.message).must_match(/Sudo failed: sudo: no tty present and no program specified/)
```

Comment:
it would be good to verify the `reason` field here and below too

Reply:
ohh ok,  will make that change

Reply:
I have replaced message with reason in most test cases . but in few cases the `_err.reason` is coming to be blank string.I kept `_err.message` in those cases